zulipterminal/ui_tools/buttons.py

```diff
@@ -81,7 +81,7 @@ def update_widget(
         if count_text[1]:
             suffix = [" ", count_text, " "]
         else:
-            suffix = ["  "]
+            suffix = [" "]
```

Comment:
The combination of this and the increase of only 1 in the left panel width, means that only fully-read topics return to the original length, so I'd suggest just using 29 for the width.

The other side effect of this change, which we may have discussed in another PR is that longer topics often show an ellipsis in the digits column, which is somewhat unclear when aligned with unread counts.

Reply:
When the count is double digit then this is going to happen anyway right? Atleast a letter doesn't come below the count digit.